src/test_l1_dsFPD.c

```diff
@@ -2932,8 +2941,8 @@ static UT_test_suite_t * pSuite7Segment = NULL;
 int test_l1_dsFPD_register ( void )
 {
 	/* add a suite to the registry */
-	pSuite = UT_add_suite( "[L1 dsFPD]", NULL, NULL );
-    pSuite7Segment = UT_add_suite( "[L1 dsFPD 7Segment]", NULL, NULL );
+        pSuite = UT_add_suite( "[L1 dsFPD]", NULL, NULL );
+        pSuite7Segment = UT_add_suite( "[L1 dsFPD 7Segment]", NULL, NULL );
```

Comment:
Comment out this pSuite7Segment things, but make a statement on the top saying the LED 7 Seg APIs are now deprecated as the HAL Interfaces are now deprecated. 

Don't delete them, but just put into #if 0 and #endif.

Reply:
Done